lib/saml/health_status.rb

```diff
@@ -1,6 +1,5 @@
 # frozen_string_literal: true
 
-require 'saml/settings_service'
```

Comment:
I think this line is still necessary because we are referencing `SettingsService` below. Without the require we would need `SAML::SettingsService`

Reply:
right now `SAML::SettingsService` requires `health_status`, so it's implicitly present. Including the deleted line was causing issues via a circular dependency

Reply:
see [this commit](https://github.com/department-of-veterans-affairs/vets-api/pull/3198/commits/79c86d10214bf795cbbc65429446c13195c5a907). for some reason it wasn't causing issues with the existing files, but adding in the new SSOe classes was introducing it somehow